### PR TITLE
Teku docker compose & enviorment 

### PR DESCRIPTION
**Pull request by atif-konasl**

### CODE REVIEW

**Review by eduardxyz — APPROVED** [review]

File: l16/teku/configs/jwtsecret [hunk 1]
```diff
@@ -0,0 +1 @@
+0x72ac4df066e0fbaa046bc9b29d60b2a3041b20e3eb5f95a991366917b8d721e6
```
Comment (NandoTheessen) [inline l16/teku/configs/jwtsecret:1]: This should not be here

File: l16/teku/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
+version: "3.4"
```
Comment (NandoTheessen) [inline l16/teku/docker-compose.yml:1]: Could you please move this to l16/docker/docker-compose.teku.yml?

File: l16/teku/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      --ethstats "${NODE_NAME}@${ETH_STATS_ADDRESS}"
+      --miner.gaslimit 60000000
+      --authrpc.jwtsecret /configs/jwtsecret
+    network_mode: host
+
+  teku_beacon:
+    image: consensys/teku:$TEKU_BEACON_VERSION
+    container_name: teku_beacon
+    depends_on:
+      - geth
```
Comment (NandoTheessen) [inline l16/teku/docker-compose.yml:47]: Please add eth2stats-client here

File: l16/teku/.env [hunk 4]
```diff
@@ -0,0 +1,26 @@
+##############################################################################################################################
```
Comment (NandoTheessen) [inline l16/teku/.env:1]: It feels like this doesn't need to be here, right? we have the default.env already in the docker folder?

File: l16/teku/configs/config.yaml [hunk 5]
```diff
@@ -0,0 +1,70 @@
+# Extends the mainnet preset
```
Comment (NandoTheessen) [inline l16/teku/configs/config.yaml:1]: Same here, feels like a duplicate

**Review by NandoTheessen — CHANGES_REQUESTED** [review]
Could you please remove the duplicated files and move the docker-compose as I outlined in the comment? :)

File: l16/docker/default.env [hunk 6]
```diff
@@ -17,3 +17,5 @@ PRYSM_BEACON_VERSION="v0.2.2-dev"
 PRYSM_BOOTSTRAP_NODE=""
 PRYSM_VERBOSITY="info"
 VALIDATOR_DATA_VOLUME="./data/validator_data"
+TEKU_BEACON_VERSION="latest"
```
Comment (NandoTheessen) [inline l16/docker/default.env:20]: Could you please pin this to whatever is the latest right now?

**Review by NandoTheessen — CHANGES_REQUESTED** [review]

File: l16/docker/default.env (hunk 6 above)
Comment (atif-konasl) [reply l16/docker/default.env:20]: latest release is:`22.6.1`
Would i add this release tag as comment in `default.env`?

Comment (NandoTheessen) [reply l16/docker/default.env:20]: Yes please, that way we won't have any new validators out there that all of a sudden run 27.0.0 that contains breaking changes :)

**Review by NandoTheessen — CHANGES_REQUESTED** [review]

File: l16/docker/default.env (hunk 6 above)
Comment (atif-konasl) [reply l16/docker/default.env:21]: Thanks for your clarification 👍

**Pull request merged**